CHANGELOG.rst

```diff
@@ -14,6 +14,11 @@ Change Log
 Unreleased
 ----------
 
+[0.26.3] - 2017-03-02
```

Comment:
This would be a new version - 0.27.0 likely. However, I generally try to avoid bumping this until right before merge to avoid merge conflicts from other PRs.

We only do revision version bumps for bugfixes; everything else is a minor version.